proxy/.solcover.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
     ],
     providerOptions: {
         default_balance_ether: 3000, //000000000000000000,
-        callGasLimit: 10000000,
-        gasLimit: 0xfffffffffff,
+        callGasLimit: "10000000",
```

Comment:
Maybe let's add "" everywhere in config files? (.solcover.js, truffle-config.js)

Reply:
I did but not everything supports value in string.